### PR TITLE
Form: update story from knobs to controls

### PR DESCRIPTION
**Pull request by thisisobate**
**What this PR does / why we need it**:
This PR updates story from knobs to controls.
**Which issue(s) this PR fixes**:

Relates to #30182

### CODE REVIEW

**Review by hugohaggmark — APPROVED** [review]
Looks good and works as expected 🎉 !

File: packages/grafana-ui/src/components/Forms/Form.story.tsx [hunk 1]
```diff
@@ -145,25 +148,11 @@ export const DefaultValues = () => {
...
-      <Button
-        onClick={() => {
-          setDefaultsIdx((s) => (s + 1) % 2);
-        }}
-        variant="secondary"
-      >
-        Change default values
-      </Button>
-    </>
-  );
```
Comment (Clarity-89) [inline packages/grafana-ui/src/components/Forms/Form.story.tsx:162]: Is there a reason to remove this? If yes, then `defaultValues` array should probably be updated as well.

Comment (thisisobate) [reply packages/grafana-ui/src/components/Forms/Form.story.tsx:162]: yeah...noticed the `change default values` button doesn't have any effects on the form. The change in the state doesn't cause any re-rendering thus making the button ineffective.

Would proceed to remove the second object in the `defaultValues `array.
Thanks! @Clarity-89 🙌

**Review by Clarity-89 — APPROVED** [review]
Good work! 🎉

**Pull request merged**